index.js

```diff
@@ -50,7 +59,21 @@ function getAllMovieTitles() {}
  *  checkIfAnyMovieHasRating(movies, "R");
  *  //> false
  */
-function checkIfAnyMovieHasRating() {}
+function checkIfAnyMovieHasRating(movies,rating = 
+  "G") {
+  if (movies.length === 0) {
+    throw "There Are No Movies Available"
+}
+return movies.some((movie) => {
+  if (
+  movie.rated === rating){
+    return true 
+  }else return false
```

Comment:
Try to follow the rule that `if` and `else` statements should have code blocks even for single statements.